### PR TITLE
Updated the code

### PR DESCRIPTION
**Pull request by aman566**
Updated the code so that the created wordlist.txt file can be in use to this code. I have created the wordlist.txt file in the same directory. @shaddygarg sir, It will definitely improve the recon.

#### Checklist
- [X] I have followed the [Contributor Guidelines](https://github.com/zdresearch/OWASP-Nettacker/wiki/Developers#contribution-guidelines).
- [X] I have added the relevant documentation.
- [X] My branch is up-to-date with the Upstream master branch.

#### Your development environment
- OS: `Linux`
- OS Version: `Ubuntu`
- Python Version: `3.x`

### CODE REVIEW

File: lib/scan/dir/engine.py [hunk 1]
```diff
@@ -251,3 +240,5 @@ def start(target, users, passwds, ports, timeout_sec, thread_number, num, total,
         os.remove(thread_tmp_filename)
     else:
         warn(messages(language, "input_target_error").format('dir_scan', target))
+
```
Comment (Ali-Razmjoo) [inline lib/scan/dir/engine.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065631)

**Comment by Ali-Razmjoo** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones removed
==============
+ lib/scan/dir/engine.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065631&bid=16668224)

File: lib/scan/dir/engine.py (hunk 1 above)
Comment (aman566) [reply lib/scan/dir/engine.py]: I have updated the code

**Comment by Ali-Razmjoo** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ lib/scan/dir/engine.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065631&bid=16668224)

**Comment by shaddygarg** [comment]
Hey @aman566, this will require a wordlist.txt to be there in the same directory. I don't think that it is suitable and please fill out your PR column by agreeing to the terms and specifying the OS and python version you tested your changes on.

**Comment by Ali-Razmjoo** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ lib/scan/dir/engine.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/zdresearch/OWASP-Nettacker/pullRequest?prid=5065631&bid=16668224)

**Comment by shaddygarg** [comment]
@Ali-Razmjoo, do we want to do this kind of thing when we already support custom wordlists and all?

**Comment by aman566** [comment]
Sir the custom wordlist is not sufficient. This wordlist will give larger attack surface. Please check and let me know for any issues :)

**Comment by aman566** [comment]
@shaddygarg Sir the custom wordlist is not sufficient. This wordlist will give larger attack surface. Please check and let me know for any issues :)

**Comment by Ali-Razmjoo** [comment]
it's always good to have a bigger wordlist, but we don't keep such a thing in the tool since we have this feature.

```
  --method-args METHODS_ARGS
                        enter methods inputs, example: ftp_brute_users=test,ad
                        min&ftp_brute_passwds=read_from_file:/tmp/pass.txt&ftp
                        _brute_port=21
```

you can simply add `--method-args dir_scan_list=read_from_file:/tmp/directories.txt` and load your files. I think it's unnecessary, and that short array is just default values from modules...

**Comment by aman566** [comment]
Totally agreeing with you but we can have it as a default wordlist, meaning if someone don't want to have its own that nettacker will search with thos one :). Any suggestions?

**Comment by aman566** [comment]
Closing this PR for clean commit history, as fixed in #272

**Pull request closed without merging**